jwala-common/src/main/java/com/cerner/jwala/common/rule/group/GroupNameRule.java

```diff
@@ -16,8 +17,43 @@ protected MessageResponseStatus getMessageResponseStatus() {
         return FaultType.INVALID_GROUP_NAME;
     }
 
+
+    protected MessageResponseStatus getMessageResponseGroupLengthStatus() { return FaultType.GROUP_NAME_TOO_LONG; }
```

Comment:
Rename to something simpler, the return type is already MessageResponseStatus so no need to repeat it in the method name. Something like, getLongNameStatus().

Also, where is this method called from? FaultType is a public enum, do we really need a method wrapped around it?